project-config.ts

```diff
@@ -501,7 +501,7 @@ export default {
       pages: [
         {
           title: "Welcome",
-          href: "/platform"
+          href: "/platform/welcome"
```

Comment:
Fix a hierarchical issue in navigation. Earlier every platform page had "Welcome" highlighted on the sidebar. This fixes that.